python/rascaline-torch/rascaline/torch/utils/power_spectrum.py

```diff
@@ -58,6 +58,9 @@ class PowerSpectrum(torch.nn.Module, PowerSpectrumBase):
 
     :param calculator_1: first calculator
     :param calculator_1: second calculator
+    :param species: List of `species_neighbor` to fill all blocks with. This option
+        might be useful when joining along the ``sample`` direction after computation.
+        If :py:obj:`None` blocks are filled with `species_neighbor` from all blocks.
```

Comment:
I feel like this is a bit clearer, what do you think?

```suggestion
    :param species: List of `species_neighbor` to use in the properties of the output. 
        This option might be useful when running the calculation on subset of the whole dataset and trying to join along the ``sample`` dimension after the calculation.
        If :py:obj:`None` blocks are filled with `species_neighbor` found in the systems.
```

Reply:
Yes I agree. Thanks!